clients/drcachesim/tracer/tracer.cpp

```diff
@@ -2987,6 +3079,32 @@ event_thread_exit(void *drcontext)
 static void
 event_exit(void)
 {
+#ifdef BUILD_PT_TRACER
+    if (op_offline.get_value() && op_enable_kernel_tracing.get_value()) {
+        drpttracer_exit();
+        /* Dump kcore and kallsyms to {kernel_pt_logsubdir}. */
+#    define SHELLSCRIPT_FMT                                                          \
+        "perf record --kcore -e intel_pt/cyc,noretcomp/k echo '' >/dev/null 2>&1 \n" \
+        "chmod 755 -R perf.data \n"                                                  \
+        "cp perf.data/kcore_dir/kcore %s/ \n"                                        \
+        "cp perf.data/kcore_dir/kallsyms %s/ \n"                                     \
+        "chmod 755 -R %s \n"                                                         \
+        "rm -rf perf.data \n"
+#    define SHELLSCRIPT_MAX_LEN 512 + MAXIMUM_PATH * 2
+        char shellscript[SHELLSCRIPT_MAX_LEN];
+        dr_snprintf(shellscript, BUFFER_SIZE_ELEMENTS(shellscript), SHELLSCRIPT_FMT,
+                    kernel_pt_logsubdir, kernel_pt_logsubdir, logdir);
```

Comment:
Why is there a "log" in the identifier? It makes it seem like it's for the debug logfiles.

Reply:
We use logsubdir to represent the directory to store user space raw traces . Do we need to follow the same style?